src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,6 +59,9 @@
 
     /**
      * Test body.
+     * @todo: #120 move constants in this file to their tests because tests
```

Comment:
@westonized I have an idea, can we add in this puzzle also note to make sure that puzzle format is verified on Travis? Currently it is not... Bulid is green but we wouldn't be able to merge this PR.